### PR TITLE
Message added

### PR DESCRIPTION
**Pull request by villenavg**

### CODE REVIEW

File: OWDTestToolkit/apps/loop.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def __init__(self, parent):
         self.market_url = "https://owd.tid.es/B3lg1r89n/market/appList.html"
         self.loop_dir = self.UTILS.general.get_config_variable("install_dir", "loop")
         self.phone_number = self.UTILS.general.get_config_variable("phone_number", "custom")
+        self.not_a_loop_user = _("is not a Firefox Hello user yet. No problem! Just create a room and share it with him.")
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/loop.py]: Where is the user name?

Comment (villenavg) [inline OWDTestToolkit/apps/loop.py]: Look at the tests

Comment (FranciscoCano) [inline OWDTestToolkit/apps/loop.py]: :+1:

**Comment by FranciscoCano** [comment]
LGTM

**Pull request merged**